src/lib/ext-activitylog.js

```diff
@@ -15,8 +15,14 @@ class Model {
     this.logs.push(...logs);
   }
 
-  setFilter({ logKey, valueEquals }) {
-    this.filter[logKey] = valueEquals;
+  /**
+   * Sets the values of any key in filter object
+   * @param {string} logKey - The key in the filter object.
+   * @param {Set} filterDescriptor - It contains the active checkboxes.
+   */
+
```

Comment:
Remove blank line between JSDoc comment and the function.

Reply:
done.